dockerfiles/base/Dockerfile

```diff
@@ -61,9 +61,13 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3${MLRUN_ANACONDA_
 
 ENV PATH /opt/conda/bin:$PATH
 
+RUN conda install -y -n base conda-libmamba-solver && \
+    conda config --set solver libmamba && \
+    conda clean -aqy
+
```

Comment:
Can we update miniconda for python 3.9? to get latest python 3.9 version

Reply:
The base image is not latest python 3.9 either. Its python 3.9.7 (iguazio python slim). May need to upgrade that 1st